pwnlib/fmtstr.py

```diff
@@ -886,7 +886,7 @@ def _leaker(self, addr):
         })
 
         leak = self.execute_fmt(fmtstr)
-        leak = re.findall(br"START(.*)END", leak, re.MULTILINE | re.DOTALL)[0]
+        leak = re.findall(br"START(.*?)END", leak, re.MULTILINE | re.DOTALL)[0]
```

Comment:
Are you sure that `END` cannot legitimately appear in the output here? I think we should not do it here. The other place is totally fine.
```suggestion
        leak = re.findall(br"START(.*)END", leak, re.MULTILINE | re.DOTALL)[0]
```